.github/workflows/build.yml

```diff
@@ -4,13 +4,14 @@ on:
   push:
     branches: [ master ]
 
-env:
-  FOUNDRY_PROFILE: ci 
-
 jobs:
   tests:
     runs-on: ubuntu-latest
     steps:
+      - name: Environment setup
+        env:
+          FOUNDRY_FUZZ_RUNS: 50000
```

Comment:
I think that it applies only to the step where it's defined, so it needs to be added to `dapp test`